app.test.js

```diff
@@ -272,4 +272,27 @@ describe('Server', () => {
         expect(errMsg).toEqual(expectedResponse)
       });
     });
+
+    describe('GET /api/v1/projects/:id/palettes', () => {
```

Comment:
Thanks for adding our final endpoint

Reply:
Yeah this will allow us to query for a specific color